src/adaptors/alpaca-finance/index.js

```diff
@@ -0,0 +1,71 @@
+const axios = require("axios");
+const utils = require('../utils');
+
+function formatSymbol(sourceName) {
+    const space = sourceName.indexOf(' ');
+    return`${sourceName.substring(space + 1)} (${sourceName.substring(0, space)})`;
+};
+
+async function apy(chain) {
+    const response = (await axios.get(
+        `https://alpaca-static-api.alpacafinance.org/${chain}/v1/landing/summary.json`
+        )).data.data;
+
+    const filteredStakingPools = response.fairLaunchStakingPools
+        .filter(p => !p.key.includes('debt'))
+    const fairLaunchStakingPools = filteredStakingPools.map(p => ({
+        pool: `${p.stakingToken.address}-staking`,
+        chain: chain == 'bsc' ? 'Binance' : utils.formatChain(chain),
+        project: 'alpaca-finance',
+        symbol: utils.formatSymbol(p.symbol),
+        tvlUsd: Number(p.tvl),
+        apy: Number(p.apy)
+    }));
+
+    const farmingPools = response.farmingPools.map(p => ({
+        pool: `${p.key}-farming-pool`,
+        chain: chain == 'bsc' ? 'Binance' : utils.formatChain(chain),
+        project: 'alpaca-finance',
+        symbol: formatSymbol(p.sourceName),
+        tvlUsd: Number(p.tvl),
+        apy: Number(p.totalApy / p.leverage)
```

Comment:
i think we need to use `(p.farmRewardApr + p.tradingFeeApr) / p.leverage` instead of `p.totalApy`.
at least when checking on the frontend it seems `totalApy` includes additional rewards which are not provided if leverage is 1x. So we can't scale it down like that.

using `(p.farmRewardApr + p.tradingFeeApr) / p.leverage` should then match exactly
whats being shown as `Total APR` like here (this is for the BNB-USDT pool on pancake swap):

<img width="658" alt="Screenshot 2022-05-12 at 10 41 44" src="https://user-images.githubusercontent.com/74533329/168041779-c543b7db-c126-445c-9084-3cedab198b3a.png">
 
In this case we will show the correct 1x leverage APR, not APY. I'm not sure how they scale it up to APY in their case tbh, but i think its fine showing the APR here especially as its only slighly below APY.

Reply:
btw, to clarify, this is only relevant for the farming pools
i ran it locally, this seems to be good:
```
apy: (Number(p.farmRewardApr) + Number(p.tradingFeeApr)) / Number(p.leverage),
```